packages/builder/babel/presets.js

```diff
@@ -1,27 +1,6 @@
-/* eslint-disable no-console */
-const path = require('path');
-
 const presets = [
   require.resolve('@babel/preset-env'),
   require.resolve('@babel/preset-react'),
 ];
 
-// Try to load `@theforeman/env/babel` from the consumer
```

Comment:
Once we will add this change, the tests in core and in plugins would break.
Please mark the commit with a `BREAKING CHANGE` status